services/cd-service/pkg/cmd/server.go

```diff
@@ -213,43 +213,44 @@ func RunServer() {
 					MigrationsPath: c.DbMigrationsLocation,
 				}
 			} else if c.DbOption == "sqlite" {
-				handler = repository.DBHandler{
+				dbCfg = repository.DBConfig{
 					DbHost:         c.DbLocation,
-					DbPort:         "",
+					DbPort:         c.DbAuthProxyPort,
 					DriverName:     "sqlite3",
-					DbName:         "",
-					DbPassword:     "",
-					DbUser:         "",
+					DbName:         c.DbName,
+					DbPassword:     c.DbUserPassword,
+					DbUser:         c.DbUserName,
 					MigrationsPath: c.DbMigrationsLocation,
 				}
 			} else {
 				logger.FromContext(ctx).Fatal("Database was enabled but no valid DB option was provided.")
 			}
-			db, err := handler.GetDBConnection()
+			db, err := repository.Connect(dbCfg)
 			if err != nil {
 				logger.FromContext(ctx).Fatal("Error establishing DB connection: ", zap.Error(err))
 			}
```

Comment:
We need to close the connection if the ping fails for whatever reason, the previous version did not do this either, but maybe a defer db.DB.close()